okhttp/src/main/kotlin/okhttp3/OkHttpClient.kt

```diff
@@ -743,6 +768,10 @@ open class OkHttpClient internal constructor(
         level = DeprecationLevel.ERROR
     )
     fun sslSocketFactory(sslSocketFactory: SSLSocketFactory) = apply {
+      check(trustManagerOverrides == null) {
```

Comment:
Mostly curious – is this an intrinsic limitation or just something to implement?